tests/common.py

```diff
@@ -978,10 +978,12 @@ def assert_setup_component(count, domain=None):
     """
     config = {}
 
-    async def mock_psc(hass, config_input, integration):
+    async def mock_psc(hass, config_input, integration, raise_on_failure=True):
```

Comment:
Why do we add a `raise_on_failure` parameter which is not used anywhere?

Reply:
It is passed through async_process_component_config at line 984